util/util.go

```diff
@@ -933,6 +960,18 @@ func GenerateDiskConfig(path string) (*DiskConfig, error) {
 	if _, err := nsExec.Execute("ls", []string{filePath}); err == nil {
 		return nil, fmt.Errorf("disk cfg on %v exists, cannot override", filePath)
 	}
+
+	// Cleaning up disk cfg file and replica subdirectory if anything
+	// goes wrong
+	defer func() {
+		if err != nil {
+			if derr := DeleteDiskPathReplicaSubdirectoryAndDiskCfgFile(path); derr != nil {
+				err = errors.Wrapf(err, "Cleaning up disk config failed err %v with path %v", derr, path)
```

Comment:
`err = errors.Wrapf(err, "cleaning up disk config path %v failed with error:  %v", path, derr)`

Always put related error at the end so it's easy to notice.

Also, don't capitalize the first character in error message, there can be other errors wrap on it too.